gamebook/option.py

```diff
@@ -1,20 +1,27 @@
 class Option(object):
-    def __init__(self, text, key, decision):
-        self.text = text
+    def __init__(self, text, key, nextscene):
+        self.text = text 
         self.key = key
-        self.decision = decision
+        self.nextscene = nextscene
 
     def show_text(self):
         return self.text
 
-    def get_decision(self):
-        return self.decision.get_decision_name()
+    def show_nextscene(self):
+        nextscene_list = 'next scenes:'
+        for i in self.nextscene:    
+            scene = i.get_scene_name()
+            rate = i.get_success_rate()
+
+            nextscene_list +=f"{scene}, {rate}\n"
+        
+        return nextscene_list
 
     def show_keys(self):
         key_list = "available keys: "
```

Comment:
What do you mean by "keys"?

Reply:
keys is the inputs the user gives when he wants to choose the option.
for example: to choose the option: "go left" the user can enter in the terminal: L or left
I should explain that with a docstring...

Reply:
Maybe `show_input_options`  + doc string?

Reply:
ok, should i change key's name as well?

Reply:
yes